### ISSUE TITLE
overfitting

### ISSUE THREAD

**Issue by faceItCalmly** [KeiLongW/battery-state-estimation#14]
Hello, I used your code to run an experiment, but I encountered overfitting. The trained network performs well on the training set but very poorly on the test set. Have you ever encountered this situation? I ran the experiment in experiments/lg/lstm_soc_percentage_lg.ipynb.

**Comment by KeiLongW** [comment]
I suggest you to start with some common solution on overfitting, such as dropout, regularization, etc.
